docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,390 @@
+
+Below is a snippet that represents our time server `package.json` (a version `4.0` package). This JSON has only the mandatory fields configured. As this is our first version, we fill the version field to be 1.0.0
+
+```
+{
+  "packagingVersion": "4.0",
+  "name": "time-server",
+  "version": "1.0.0",
+  "maintainer": "https://github.com/mesosphere/universe",
+  "description": "This is a simple Python HTTP server that displays a webpage that says the current time at the server location",
+  "tags": ["python", "http", "time-server"]
+}
+```
+
+Note that the version field specifies the version of the package and this is independent of the directory number inside the `time-server` directory.
+
+You can read more about the various fields in this field [here](https://github.com/mesosphere/universe#configjson) or can see [`repo/meta/schema/package-schema.json`](repo/meta/schema/package-schema.json) for the full JSON schema outlining what properties are available for each corresponding version of a package.
+
+
+#### marathon.json.mustache
+This file is a [mustache template](http://mustache.github.io/) that when rendered will create a
+[Marathon](http://github.com/mesosphere/marathon) app definition capable of running your service. The first level of validation is that after Mustache substitution, the result must be a JSON document. Once the JSON document is produced, it will be valid request body for Marathon's `POST /v2/apps` endpoint ([Marathon API Documentation](https://mesosphere.github.io/marathon/docs/rest-api.html)).
+
+This is the Marathon file that we would use :
+
+```
+{
+  "id": "{{service.name}}",
+  "cpus": {{service.cpus}},
+  "mem": {{service.mem}},
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{resource.assets.container.docker.timeserverimage}}",
+      "network": "HOST"
+    }
+  },
+  "portDefinitions": [
+    {
+      "port": 0,
+      "protocol": "tcp"
+    }
+  ]
+}
+```
+
+The service `name`, `cpus` and `mem` are populated from the `config.json` file. The image is populated from the `resource.json` file. We are using HOST mode of networking to dynamically get a port from the available pool. Read [about Marathon ports](https://mesosphere.github.io/marathon/docs/ports.html) to understand modes in detail.
+
+
+### Step 4 : Testing the package
+Now that you have the package built, we need to make sure everything works as expected before publishing to the community.
+
+
+#### Validation using build script.
+You can execute the script inside the `scripts/build.sh` to make sure all the JSON schema comply to specifications and to install any missing libraries. This script is also executed as a precommit hook.
+
+It may throw some error if there are any unrecognized fields in the package files. Fix those error and re-execute the command until the build is successful.
+
+Now, we can run the Universe server locally to test and install our package.
```

Comment:
By "run ... locally", did you really mean "build locally, and test in our DC/OS cluster"?